prow/config.yaml

```diff
@@ -3535,6 +3535,47 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+   - name: pull-kubernetes-e2e-gci-gke
```

Comment:
This is also under the config for kubernetes/test-infra, kubernetes/kuberenetes is configured in a different section.